src/utilities/project-utils.ts

```diff
@@ -10,109 +10,231 @@ import pcLogo from "../assets/project-images/pc-logo.png";
+    footerLogo:[ nsfLogo, concordLogo, stemCreateLogo ],
+  },
+  {
+    id: 22,
+    name: "Measuring Collaboration",
+    projectLogo: undefined,
+    url: "",
+    footer: "<p>This activity was developed under a grant from the&nbsp;<a title=\"nsf.gov - National Science Foundation - US National Science Foundation (NSF)\" href=\"http://nsf.gov/\">National Science Foundation</a>&nbsp;(DUE-1535224).</p>",
+    footerLogo:[ nsfLogo ],
+  },
+  {
+    id: 23,
+    name: "Evolution Readiness",
+    projectLogo: undefined,
+    url: "",
+    footer: "<p>This material is based upon work supported by the National Science Foundation under Grant No. DRL-0822213. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation.</p>",
+    footerLogo:[],
+  },
+  {
+    id: 24,
+    name: "InSPECT",
+    projectLogo: undefined,
+    url: "",
+    footer: "<p>These <a title=\"The Concord Consortium: InSPECT\" href=\"http://concord.org/inspect\" target=\"\">InSPECT</a>&nbsp;materials were developed and researched with funding from the&nbsp;<a title=\"National Science Foundation\" href=\"http://nsf.gov/\" target=\"\">National Science Foundation</a>&nbsp;(DRL-1640054).</p><p>This open educational resource from&nbsp;<a title=\"The Concord Consortium\" href=\"http://concord.org/\" target=\"\">The Concord Consortium</a>&nbsp;is free for use under the&nbsp;Creative Commons Attribution International license (<a title=\"\" href=\"http://creativecommons.org/licenses/by/4.0/\" target=\"\">CC-BY 4.0</a>) and powered by open source software packages. When sharing this resource, please include attribution to the Concord Consortium and links to&nbsp;<a title=\"\" href=\"http://concord.org/\" target=\"\">concord.org</a>&nbsp;and the CC-BY-4.0 license (see&nbsp;<a title=\"\" href=\"http://concord.org/licenses\" target=\"\">licensing details</a>). Copyright 2018 The Concord Consortium.</p><p><br />Questions? Comments? You can reach us at&nbsp;<a href=\"mailto:inspect@concord.org\">inspect@concord.org</a>.</p>",
+    footerLogo:[ concordLogo, nsfLogo ],
+  },
+  {
+    id: 25,
+    name: "CodeR4MATH",
+    projectLogo: undefined,
+    url: "",
+    footer: "<p class=\"footer-txt\" style=\"color: #555;\">Copyright &copy; 2020 <a href=\"https://concord.org/\">The Concord Consortium</a>. All rights reserved. This activity is licensed under a <a href=\"https://creativecommons.org/licenses/by/4.0/\"> Creative Commons Attribution 4.0 License </a>. The software is licensed under <a href=\"https://opensource.org/licenses/BSD-2-Clause\"> Simplified BSD </a>, <a href=\"https://opensource.org/licenses/MIT\">MIT</a> or <a href=\"https://opensource.org/licenses/Apache-2.0\">Apache 2.0</a> licenses. Please provide attribution to the Concord Consortium and the URL <a href=\"https://concord.org\">https://concord.org</a>.</p>",
+    footerLogo:[],
```

Comment:
I didn't see these additional projects when I was making the header.  Where did you find them?  Production only?

Reply:
I took from here https://authoring.concord.org/projects
Not sure how many of them are actually being used, but I thought to add them just because they exist.

Reply:
Ah, good thinking!  We'll probably need them eventually.